modules/ROOT/partials/configuration/advtemplate_list.adoc

```diff
@@ -0,0 +1,75 @@
+[[advtemplate_list]]
+== `advtemplate_list`
+
+This option retrieves the list of available Templates.
+
+*Type:* see the example below
+
+*Possible values:* see the example below
+
+=== Example: using `advtemplate_list`
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  advtemplate_list: () => {
+    return Promise.resolve(
+    [
+      {
+        id: '1',
+        title: 'Template 1'
+      },
+      {
+        id: '2',
+        title: 'Category 2',
+        items: [
+      {
+        id: '3',
+        title: 'Template 2.1',
+      },
+      {
+        id: '4',
+        title: 'Template 2.3',
+      }
+    ]
+  }
+]
+    );
+});
+----
```

Comment:
```suggestion
tinymce.init({
  selector: 'textarea',  // change this value according to your html
  advtemplate_list: () => {
    return Promise.resolve(
    [
      {
        id: '1',
        title: 'Resolving tickets',
        content: '<p>As we have not heard back from you in over a week, so we have gone ahead and resolved your ticket</p>'
      },
      {
        id: '2',
        title: 'Quick replies',
        items: [
          {
            id: '3',
            title: 'Message received',
            content: '<p>Just a quick note to say we have received your message, and will get back to you within 48 hours.</p>'
          },
          {
            id: '4',
            title: 'Progress update',
            content: '</p>Just a quick note to let you know we are still working on your case</p>'
          }
        ]
      }
    ]
  );
});
----
```
Fixed indentation and added some sample content since 'content' is required

Reply:
Actually the`content` is not required in this response. List response is used for creating the list of items for the Tree component. In order to keep traffic small the `content` property is removed from the list response

Reply:
> Actually the `content` is not required in this response.

A new [edit](https://github.com/tinymce/tinymce-docs/commit/f7f051e048e6300bb2f55cdcb81151f51b7fe996) that notes this.